### ISSUE TITLE
Angular 16 not detected by CT

### ISSUE THREAD

**Issue by MikeMcC399** [cypress-io/cypress#26852]
### Current behavior

After Angular 16 has been installed, the Cypress desktop app in Component Testing mode does not detect the installation and shows the message "Waiting for you to install the dependencies".

![Angular 16](https://github.com/cypress-io/cypress/assets/66998419/985c35d0-ccb3-4890-989c-0c2e4d2afa75)

### Desired behavior

Angular 16 should be detected in Component Testing. The Cypress [12.11.0](https://docs.cypress.io/guides/references/changelog#12-11-0) changelog specifically mentions this as:

- Added Component Testing support for Angular 16. Addresses #26044.

### Test code to reproduce

```bash
mkdir cy-angular-ct
cd cy-angular-ct
git init
npm init -y
npm install cypress@12.13.0 -D -E
npx cypress open
```

Click on "Component Testing"
Select Front-end framework `Angular`
Click "Next step"
Copy and paste the command as instructed:
`npm install -D @angular/cli @angular-devkit/build-angular @angular/core @angular/common @angular/platform-browser-dynamic`
*Note: The spinner and the text "Waiting for you to install the dependencies" remain displayed*

### Cypress Version

12.13.0

### Node version

18.16.0

### Operating System

Ubuntu 22.04

### Other

- It looks like a similar root cause to https://github.com/cypress-io/cypress/pull/26815 concerning putting the version in single quotes.

**Comment by MikeMcC399** [comment]
It looks like each line 

`  minVersion: '^=13.0.0 || ^=14.0.0 || ^=15.0.0' || '^=16.0.0',`

should be

`  minVersion: '^=13.0.0 || ^=14.0.0 || ^=15.0.0 || ^=16.0.0',`

https://github.com/cypress-io/cypress/blob/d91177a0f343c14f42b3cd786d7ae45289496ae9/packages/scaffold-config/src/dependencies.ts#L100-L143

**Comment by MikeMcC399** [comment]
@jordanpowell88 

Do you want a PR for this issue? I have a [fix branch](https://github.com/MikeMcC399/cypress/tree/issue-26852-angular-16-ct-detection) prepared if you would like it.
Would it be too late for Cypress `12.13.1` or is there still time?

**Comment by jordanpowell88** [comment]
@MikeMcC399 here is my PR that should get this released next week https://github.com/cypress-io/cypress/pull/26908

**Comment by MikeMcC399** [comment]
@jordanpowell88 

> here is my PR that should get this released next week #26908

I somehow missed your new PR. It's very good you have one prepared anyway!

I was really just practising because although I have submitted a lot of other PRs into the Cypress eco-system I had never done one for the core Cypress product, so at least I have now done a dry run of preparation and testing ready for some other future PR! I'll delete my branch now as it is more or less a duplicate of yours (except I added to 12.13.1 and yours is for 12.14.0 in the changelog).
